src/dashboardPage/RidesListItem.js

```diff
@@ -1,5 +1,5 @@
 import React, {useState} from 'react'
```

Comment:
I would give the file "RidesListItem.js" and the component "ListItem" the same name.